src/components/EpisodeCard.tsx

```diff
@@ -0,0 +1,38 @@
+import IEpisode from "../interfaces/IEpisode";
+import cleanString from "../util/cleanString";
+import padNumber from "../util/padNumber";
+import "./episodeCard.css";
+
+interface EpisodeCardProps {
+    key: number;
```

Comment:
This is only required for the callback function for the .map inside App.tsx that uses AllEpisodes. It isn't needed as part of your props interface in this component as far as we can see.